.circleci/config.yml

```
@@ -7,7 +7,7 @@ version: 2
 jobs:
   style-check:
     docker:
-      - image: circleci/python:3.9
+      - image: circleci/python:3.10
```

Let's try migrating from legacy `circleci` to the newer `cimg` images. If it fails, we can revert that change for this PR - just trying to future-proof the CI a bit.

https://circleci.com/docs/2.0/next-gen-migration-guide/
https://hub.docker.com/r/cimg/python

```suggestion
      - image: cimg/python:3.10
```

---

Good call, forgot this shift happened.